mods/player_api/api.lua

```diff
@@ -135,3 +145,43 @@ minetest.register_globalstep(function(dtime)
 		end
 	end
 end)
+
+-- player get node vars
+local update_time = 1.0
+local timer = 0
+
+minetest.register_globalstep(function(dtime)
+
+	timer = timer + dtime
+	if timer < update_time then
+		return
+	end
+	timer = 0
+
+	local name, pos, ndef
+
+	for _,player in pairs(minetest.get_connected_players()) do
+
+		name = player:get_player_name()
+		pos = player:getpos()
+
+		pos.y = pos.y - 0.1
+		player_api.nodes[name].stand = minetest.get_node_or_nil(pos) or
```

Comment:
get_node_or_nil() must be a lot less expensive to use repeatedly than get_node(), I take it?

Reply:
It saves for an additional check if node exists, and I can use OR to fill in the gaps.